pom.xml

```diff
@@ -1506,6 +1506,7 @@
     <maven.warbucks.version>1.1.0</maven.warbucks.version>
     <os.maven.version>1.5.0.Final</os.maven.version>
     <spotbugs.version>3.1.11</spotbugs.version>
+    <surefire.version>2.22.2</surefire.version>
```

Comment:
We do not have this property before?

Reply:
We used this property but it was defined in [ASF parent pom](https://github.com/apache/maven-apache-parent/blob/apache-21/pom.xml#L93).

Reply:
OK, got it.